examples/kafka/README.md

```diff
@@ -0,0 +1,76 @@
+How to run kafka on PAI
+===
```

Comment:
pls use `#` and `##` instead of `==` and `--` for section title to keep consistent with other readme docs